.gitignore

```diff
@@ -88,3 +88,6 @@ sw.*
 
 # Vim swap files
 *.swp
+
+# Husky config files
+.husky
```

Comment:
This shouldn't be here

Reply:
Husky generated a husky.sh file in that folder, do we really want that to be committed?

Reply:
Just rebase from master :)

Reply:
The shell file shouldn't be committed, but there is a .gitignore inside husky that should tell git to ignore it

Reply:
For some reason it committed to me. Now it doesn't, so I'm removing this. Strange!

Reply:
Ah no worries